qiita_db/study.py

```diff
@@ -206,6 +207,26 @@ def get_by_status(cls, status):
 
         return studies
 
+    @staticmethod
+    def get_by_portal(portal):
```

Comment:
Is not only this function. I disagree with other parts. With the add/remove portal functions each portal can give itself access to all the studies; which I disagree. Only QIITA should be able to change the portal - this way Qiita becomes also de admin part of the database.

The EMP portal should not know that there are other portals. It only sees part of the data and it has the illusion that everything that it can see is the entire DB.